### ISSUE TITLE
chore: add `తెలుగు - te-IN` language support

### ISSUE THREAD

**Issue by brionmario** [asgardeo/javascript#196]
### Current Limitation

Currently, the Asgardeo JavaScript SDK does not have support for **Telugu (తెలుగు - te-IN)**. Users who prefer Telugu will not see UI components translated, limiting accessibility and localization for Telugu-speaking users in India and globally.

> [!Important]
> Contributors for this task should be **native Telugu speakers or have professional fluency** to ensure accurate, culturally appropriate translations.

### Suggested Improvement

Add **Telugu (`te-IN`) language support** to the `@asgardeo/i18n` package.

**Implementation Plan:**

1. Create a new locale file:

    Follow https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package
    
2. Add documentation for usage:  
   - Update the SDK docs: https://wso2.com/asgardeo/docs/sdks/react/overview/

### Please select the package issue is related to

@asgardeo/i18n

### Version

current

### Reporter Checklist

- [x] I have searched the existing issues and this is not a duplicate.
- [x] I have provided all the necessary information.
- [x] I have verified the improvement is not available in the latest version of the package.

**Comment by ProgrammingPirates** [comment]
@brionmario 
I hope you are doing well. I have reviewed this issue and would like to respectfully request to be assigned to it. I am eager to contribute and will ensure that my work aligns with the project’s guidelines and standards.
can u assign me this issue pls

Thank you very much for considering my request. 🙏

**Comment by jaisheel15** [comment]
Hi @brionmario ,

I’d like to contribute Telgu (తెలుగు - te-IN )language support 


 I am a native speaker of తెలుగు  and can provide accurate, culturally appropriate translations.
Please assign me this issue .

Thanks!

**Comment by brionmario** [comment]
> [@brionmario](https://github.com/brionmario) I hope you are doing well. I have reviewed this issue and would like to respectfully request to be assigned to it. I am eager to contribute and will ensure that my work aligns with the project’s guidelines and standards. can u assign me this issue pls
> 
> Thank you very much for considering my request. 🙏

Hey @ProgrammingPirates,

Thanks a lot for the interest!
However, @mohith2883 was first to show interest here: https://github.com/asgardeo/javascript/issues/145#issuecomment-3341318172, hence i'm assigning the issue to him.

We need native speakers for our translations.
Are there any languages you currently speak that are not being worked on: [i18n Issues](https://github.com/orgs/wso2/projects/134/views/1?filterQuery=label%3A%40asgardeo%2Fi18n)

If not, have a look at the board for more issues: https://github.com/orgs/wso2/projects/134/views/1
